src/components/DurationPopover.vue

```diff
@@ -1,30 +1,62 @@
 <template>
-  <ion-select :interface-options="customPopoverOptions" interface="popover">
-    <ion-select-option value="a">Every 5 minutes</ion-select-option>
-    <ion-select-option value="b">Every 15 minutes</ion-select-option>
-    <ion-select-option value="c">Hourly</ion-select-option>
-    <ion-select-option value="d">Daily</ion-select-option>
-    <ion-select-option value="e">Disabled</ion-select-option>
+  <!-- TODO Make values dynamic and internationalise text  -->
+  <ion-select :interface-options="customPopoverOptions" interface="popover" :value="getJobStatus(id) ? getJobStatus(id) : 'SERVICE_DRAFT'" @ionChange="updateJob(id)" >
```

Comment:
> getJobStatus(id) ? getJobStatus(id) : 'SERVICE_DRAFT' 
 
Handle this in getter?

Reply:
Sir, handled this in getter to return default value.